python/tank/platform/engine.py

```diff
@@ -1774,14 +1803,26 @@ def _define_qt_base(self):
                 base["dialog_base"] = importer.QtGui.QDialog
             else:
                 base["dialog_base"] = None
-            base["wrapper"] = importer.wrapper
         except:
 
             self.log_exception("Default engine QT definition failed to find QT. "
                                "This may need to be subclassed.")
 
         return base
 
+    def __define_qt5_base(self):
+        """
+        This will be called at initialization to discover every PySide 2 modules. It should provide
+        every Qt modules available as well as two extra attributes, ``__name__`` and
+        ``__version__``, which refer to the name of the binding and it's version, e.g.
+        PySide2 and 2.0.1.
+
+        .. note:: PyQt5 not supported since it runs only on Python 3.
+
+        :returns: A dictionary with all the modules, __version__ and __name__.
+        """
+        return QtImporter(interface_version_requested=QtImporter.Qt5).base
```

Comment:
hmm, i think my above comment can be ignored. Looked at older versions of core and it imports PySide at startup, so looks like we are already doing deep loading at init.